{{cookiecutter.github_project_name}}/js/lib/labplugin.js

```diff
@@ -0,0 +1,16 @@
+var {{ cookiecutter.npm_package_name }} = require('./index');
+var base = require('@jupyter-widgets/base');
+
+module.exports = {
+  id: '{{ cookiecutter.npm_package_name }}',
+  requires: [base.IJupyterWidgetRegistry],
+  activate: function(app, widgets) {
+      widgets.registerWidget({
+          name: '{{ cookiecutter.npm_package_name }}',
+          version: {{ cookiecutter.npm_package_name }}.version,
```

Comment:
this doesn't really work well for names with `-` inside, which seems to be encouraged.

Reply:
Very good point. There's no reason that variable name can't just be `plugin` or something.